.github/scripts/check-compose-volumes.py

```
@@ -0,0 +1,98 @@
+        sys.exit(1)
+
+    if not isinstance(data, dict):
+        return []
+
+    errors = []
+
+    top_volumes = data.get("volumes")
+    if isinstance(top_volumes, dict):
+        for key in top_volumes:
+            if VOLUME_SUFFIX_RE.match(str(key)):
+                errors.append(str(key))
```

Ruff (0.15.21)

[warning] Use `list.extend` to create a transformed list

(PERF401)